web3/geth.py

```diff
@@ -0,0 +1,20 @@
+from web3.module import (
+    Module,
+)
+from web3.personal import (
+    Personal,
+)
+
+
+class GethPersonal(Personal):
+    """
+    https://github.com/ethereum/go-ethereum/wiki/Management-APIs#personal
+    """
+    def __init__(self, w3):
+        self.w3 = w3
```

Comment:
This will override the inherited Module \_\_init\_\_, which could cause a bug in the future.